ast/ddl.go

```diff
@@ -1631,6 +1632,14 @@ func (n *TableOption) Restore(ctx *RestoreCtx) error {
 		} else {
 			ctx.WritePlainf("%d", n.UintValue)
 		}
+	case TableOptionSecondaryEngine:
+		ctx.WriteKeyWord("SECONDARY_ENGINE ")
+		ctx.WritePlainf("= ")
+		if n.StrValue == "NULL" {
```

Comment:
`SECONDARY_ENGINE 'NULL'` and `SECONDARY_ENGINE NULL` are still semantically different, the former requests a secondary engine literally named "NULL".